### DISCUSSION TITLE
关于basic_transfer_test

### DISCUSSION THREAD

**Discussion by lxr12010513** [SUSTech-CS305-Fall22/CS305-Project-Skeleton#35] [Q&A]
请问basic_transfer_test一般多少秒可以跑出结果

**Comment by TANSixu** [comment]
hi，更新后的话很快 应该不到1s～

**Reply by Lynchrocket** [reply]
请问不到1s出结果是优化后的吗？我们组跑basic_transfer_test一般要3~4s。

**Reply by TANSixu** [reply]
> 请问不到1s出结果是优化后的吗？我们组跑basic_transfer_test一般要3~4s。

实现了窗口发送应该都差不多这个时间。不过不用太在意运行时间，我们只有bonus会测时间～
